content/docs/other-guides/istio-in-kubeflow.md

```diff
@@ -0,0 +1,96 @@
+different parts of the workflow. Developing a complete ML Workflow or an ML
+development environment requires combining multiple services and components.
+Kubeflow provides the underlying infrastructure and middleware setup that makes
+it possible to be able to put such disparate components together.
+
+Kubeflow uses Istio as a uniform way to secure, connect, and monitor microservices. Specifically:
+
+ - Securing service-to-service communication in a Kubeflow deployment with
+   strong identity-based authentication and authorization.
+ - A policy layer for supporting access controls and quotas.
+ - Automatic metrics, logs, and traces for traffic within the deployment
+   including cluster ingress and egress.
+
+
+## Istio in Kubeflow
+
+The following diagram illustrates how user requests interact with services in
+Kubeflow. It walks through the process when a user requests to create a new
+notebook server via the Notebooks Servers UI accessible through the Central Dashboard.
+
+
+<img src="/docs/images/Istio-in-KF.svg" 
+  alt="Select active profile "
+  class="mt-3 mb-3 border border-info rounded">
+
+  1. The user request is intercepted by an identification proxy which talks to
+     a SSO service provider such as IAM on Cloud Services Provider or Active
+     Directory/LDAP on-premises. 
+  1. Once the user is authenticated, the request is modified by the Istio
+     Gateway to include a JWT Header token containing the identity of the user.
+     All requests thorughout the service mesh carry this token along.
+  1. The Istio RBAC policies are applied on the incoming request to validate
+     the access to the service and the requested namespace. If either of those
+     are inaccessible to the user, an error response is sent back. 
+  1. If the request is validated, it is forwarded to the appropriate controller
+     (Notebooks Controller in this case).
+  1. Notebooks Controller will create the notebook pod in the namespace user requested.
+
+Further actions by the user with the notebook to create training jobs or other
+resources in the namespace go through a similar process. Profiles Controller
+manages the creation of profiles, creates and applies appropriate Istio
+policies. For more details, please see [multi-user
+overview](/docs/other-guides/multi-user-overview/).
+
+
+## Deploying Kubeflow without Isito
+
+It is not possible to deploy Kubeflow without Istio. Kubeflow needs the Istio
+CRDs to express the new route to access the created Notebook from the Gateway.
+In the future, if there is a universal way to express that (e.g. the Service
+Mesh Interface effort), Kubeflow may use that. Until then, Istio CRDs are being
+used.  If you want to use a different Service Mesh, you could make a controller
+that:
+
+  1. Watches Istio CRDs (VirtualServices, ServiceRoles, ServiceRoleBindings)
+     and extracts the necessary information from them.
+  1. Translate those into configuration for the Service Mesh of your choice.
```

Comment:
To make the bullet points gramatically parallel, change "Translate" to "Translates" here, and "Apply" to "Applies" in the next line.